### PR TITLE
responsive desing

### PR DESCRIPTION
**Pull request by joredudiaz**
## Summary

responsive design

## Jira Story

https://trello.com/c/ZVwv57W5

## Screenshot

image

### CODE REVIEW

File: mock-up/scss/titles.scss [hunk 1]
```diff
@@ -15,3 +15,12 @@
   right: 32px;
   top: 5px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .title {
+    align-items: center;
+    display: flex;
```
Comment (betoobernal) [inline mock-up/scss/titles.scss]: don't repeat.

File: mock-up/scss/tech.scss [hunk 2]
```diff
@@ -47,3 +47,28 @@
   margin: 20px 0;
   text-align: center;
 }
+
+@media only screen and (max-width: 1024px) {
+  .techs-container {
+    align-items: center;
+    display: grid;
```
Comment (betoobernal) [inline mock-up/scss/tech.scss]: same

File: mock-up/scss/styles.scss [hunk 3]
```diff
@@ -14,3 +14,12 @@
 .content {
   grid-area: content;
 }
+
+@media only screen and (max-width: 1024px) {
+  .container {
+    display: grid;
```
Comment (betoobernal) [inline mock-up/scss/styles.scss]: same

**Review by betoobernal — CHANGES_REQUESTED** [review]
image

fix this when the resolution is larger than 1024

File: mock-up/scss/footer.scss [hunk 4]
```diff
@@ -18,3 +18,17 @@
   margin: 20px 20px 0 0;
   align-self: flex-end;
 }
+
+@media only screen and (max-width: 1024px) {
+  .footer-title {
+    font-size: 16px;
+  }
+
+  .footer-logo {
```
Comment (FrankIglesias) [inline mock-up/scss/footer.scss:27]: use width and max width here

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by betoobernal — APPROVED** [review]

**Pull request merged**